LayoutTests/imported/w3c/web-platform-tests/storage-access-api/resources/sandboxed-iframe-allow-storage-access.html

```diff
@@ -29,6 +29,7 @@
     t.add_cleanup(async () => {
       await test_driver.set_permission({name: 'storage-access'}, 'prompt');
     });
+    await SetFirstPartyCookie(location.origin, "initial-cookie=unpartitioned;Secure;SameSite=None;Path=/");
```

Comment:
Looks like you could use the default value for the cookie because the test compares the cookie's value, but this is fine.